src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java

```diff
@@ -306,10 +306,16 @@ private static RawRepositories buildRawRepositories(@Nullable List<Repository> r
         return rawRepositories;
     }
 
+    private Set<Path> skipOtherMavenProjects() {
+        return mavenSession.getProjects().stream()
```

Comment:
Note for posterity:
`mavenSession#getProjects()` returns the projects that are already sorted in reactor order.

```java
    @Override
    public void execute() throws MojoExecutionException {
        if (mavenSession.getCurrentProject == mavenSession.getProjects().get(mavenSession.getProjects().size() - 1)) {
           // do something when the last project is building.
        }
    }
```